tools/org.eclipse.n4js.hlc.base/META-INF/MANIFEST.MF

```diff
@@ -19,6 +19,10 @@ Require-Bundle: org.eclipse.n4js,
  org.eclipse.n4js.external.libraries,
  org.eclipse.core.runtime,
  org.eclipse.n4js.n4jsx,
- org.eclipse.equinox.app
+ org.eclipse.equinox.app,
+ org.eclipse.n4js.generator.common,
+ org.eclipse.n4js.transpiler.es,
+ org.eclipse.n4js.generator,
+ org.eclipse.n4js.n4jsx.generator
```

Comment:
this requires update of `org.eclipse.n4js.headless.product.product`
the `org.eclipse.n4js.n4jsx.generator` needs to be added into the plugins list.

Reply:
done